### DISCUSSION TITLE
Using register contents as issue body?

### DISCUSSION THREAD

**Discussion by PhilippFeO** [pwntester/octo.nvim#574] [Q&A] [answered]
Is it possible to use register contents as body for an issue?

After typing `Octo issue create`, octo.nvim asks me, if I want to use the contents of the current buffer as body for my issue. I normally don't but a subset of my buffer.

If it is not possible to use register contents, can I at least disable the buffer content question somehow?

**Comment by williambdean** [comment]
I tend to use gitlinker: https://github.com/ruifm/gitlinker.nvim to grab the link and will just paste that into the issue after creation.

I also have a workflow where I create issue from body and title: https://github.com/wd60622/dotfiles/blob/25168a19ec5f85ed0223051d78f4d9ace729d97d/nvim/lua/plugins/version-control.lua#L7-L22

**Reply by PhilippFeO** [reply]
Just for clarification: It is not possible to feed Octo.nvim with `--title` and `--body`? I searched the docs but without matches. Sometimes, it would be quite handy to also edit the issue.

Nevertheless, your resources are sufficient enough to fullfill my needs!

**Reply by williambdean** [reply, marked as answer]
My understanding of the project, those flags **do not** exist. 
This is the function that is being used https://github.com/pwntester/octo.nvim/blob/84a4f195bca82b99d0bf1c8acf64fda00eb0a118/lua/octo/commands.lua#L782-L820
That provides the prompts if the title and body are not provided

**Reply by PhilippFeO** [reply]
Thanks for the verifying my assumption. And thank you for your advice and work!

**Reply by williambdean** [reply]
I recently added this addition to it. You can highlight a block of text then reference it in an issue: https://github.com/wd60622/dotfiles/blob/86830b50443fbab05273f52b95635593544c65fa/nvim/lua/plugins/version-control.lua#L43-L66

**Reply by PhilippFeO** [reply]
I am impressed that you remembered this thread and took the time to answer/update on your config!

Thank you for the inspiration!
